joomla/joomla-cms
# Fix submenu being (transparent) displayed despite not being 'open', blocking usage of contents at the edge of all backend screens

Pull Request for Issue #21360

### Summary of Changes
Hide sub-menus when not in use


### Testing Instructions
See #21090 
you can not use the first column at the left of the screen (or other contents at the left (right for RTL))
because (a transparent) sub-menu is open and receiving the clicks


### Expected result
The sub-menus are not open when not used


### Actual result
The sub-menus are open when not used, (and are transparent) 
blocking usage of contents at the edge of all backend screens


### Documentation Changes Required
None
